src/components/dropdowns/dropdown/Dropdown.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from "react";
+import styled from "styled-components"
+import { dropdownWrapperCSS, dropdownOuterCSS, dropdownInnerCSS, dropdownArrowCSS } from "./DropdownCSS";
+import { ChevronUp, ChevronDown } from "../../../assets/icons";
+import { colors } from "../../colors/index";
+
+const Dropdown = ({outerText, innerText, width}) => {
+	
+	const [open, setOpen] = useState(false);
+
+	const toggleDropdown = () => {
+		setOpen(!open);
+	};
+
+	const ToggleStyles = (p) => {
+		if(open) {
+			return `
+				{
+					background: ${colors.gradientMain};
+				}
+			`
+		} else {
+			return `
+				{
+					background: ${colors.white};
+				}
+			`
+		}
+	}
+
+	const DropdownWrapper = styled.div`
+		${dropdownWrapperCSS}
+		${(p) => `{ width: ${width}px }`}
+	`;
+
+	const DropdownOuter = styled.div`
+		${dropdownOuterCSS}
+		${(p) => ToggleStyles(p)}
```

Comment:
if you're not using the parameter you can just get rid of it and do this
`${ToggleStyles()}`